pipeline/src/pipeline/pipelinetypes.jl

```diff
@@ -22,3 +20,43 @@ The pipeline type for the Rt pipeline without renewal in test mode.
 """
 struct EpiAwareExamplePipeline <: AbstractEpiAwarePipeline
 end
+
+"""
+The pipeline type for the Rt pipeline for an outbreak scenario where Rt decreases
+    smoothly over time to Rt < 1.
+
+# Example
+
+```julia
+using EpiAwarePipeline, Plots
+pipeline = SmoothOutbreakPipeline()
+Rt = make_Rt(pipeline) |> Rt -> plot(Rt,
+    xlabel = "Time",
+    ylabel = "Rt",
+    lab = "",
+    title = "Smooth outbreak scenario")
+```
+"""
+struct SmoothOutbreakPipeline <: AbstractRtwithoutRenewalPipeline
+end
+
+"""
+The pipeline type for the Rt pipeline for an outbreak scenario where Rt has
+    discontinuous changes over time due to implementation of measures.
+"""
+struct MeasuresOutbreakPipeline <: AbstractRtwithoutRenewalPipeline
```

Comment:
ideally both the outbreaks would share a meta type (with the susceptible depletion and then measures would be a special case? Same in the endemic setting?

Reply:
Ultimately, the analysis is coming down to about 4/5 examples + some actual data.

In terms of readability with regard to an eventual reader of a paper, I think having more verbose types is worth it even if there are more lines of code.

On the other hand, having a powerful traits based system would be very slick for letting people define their own analysis... this is connected to #232 because a traits based system would be a natural feature of a `EpiAwarePipeline` package.

The main question is what do we prioritise first?

Reply:
I aagree we should gt it working and then think about reusability at a later date